nevergrad/optimization/es.py

```diff
@@ -32,6 +33,13 @@ def __init__(
         self._waiting: tp.List[p.Parameter] = []
         # configuration
         self._config = EvolutionStrategy() if config is None else config
+        if self._config.ranker == "simple":
+            self._ranker = None
+        elif self._config.ranker == "nsga2":
+            self._ranker = rankers.NSGA2Ranking() #Note this does not control the selection mechanism
+        else:
+            raise NotImplementedError
```

Comment:
```suggestion
        self._ranker: tp.Any = None   # TODO better typing (eventually)
        if self._config.ranker == "nsga2":
            self._ranker = rankers.NSGA2Ranking()
        elif self._config.ranker != "simple":
            raise NotImplementedError(f"Unknown ranker {self._config.ranker}")
```

Is the class adding anything to a mere function? Aka is there any sort of memoization? If not, then a function is much better, as it has a clearer API.